metron-stellar/stellar-common/src/test/java/org/apache/metron/stellar/dsl/functions/BasicStellarTest.java

```diff
@@ -218,6 +218,25 @@ public void testVariablesUsed() {
     }
   }
 
+  @Test
+  public void testConditionalsAsMapKeys() {
+    {
+      String query = "{ ( RET_TRUE() && y < 50 ) : 'info', y >= 50 : 'warn'}";
+      Map<Boolean, String> ret = (Map)run(query, ImmutableMap.of("y", 50));
+      Assert.assertEquals(ret.size(), 2);
+      Assert.assertEquals("warn", ret.get(true));
+      Assert.assertEquals("info", ret.get(false));
+    }
+  }
+
+
+  @Test
+  public void testConditionalsAsFunctionArgs() {
+    {
+      String query = "RET_TRUE(y < 10)";
+      Assert.assertTrue((boolean)run(query, ImmutableMap.of("y", 50)));
+    }
+  }
 
```

Comment:
Do we need lambda with args tests for this as well?

Reply:
Well, we know that lambdas can be arguments for functions (because of `MAP` and `REDUCE`)

Reply:
I wouldn't be opposed to adding the case, mind you.  I'll go ahead and add it to be explicit.

Reply:
I was thinking of something else, but on second thought I don't think it is valid.